### PR TITLE
librdkafka static bundle v1.5.2

### PR DESCRIPTION
**Pull request by edenhill**

### CODE REVIEW

File: kafka/build_darwin.go [hunk 1]
```diff
@@ -6,8 +6,8 @@
 package kafka
 
 // #cgo CFLAGS: -I${SRCDIR}
-// #cgo LDFLAGS: ${SRCDIR}/librdkafka/librdkafka_darwin.a  -lm -lsasl2 -lz -ldl -lpthread
+// #cgo LDFLAGS: ${SRCDIR}/librdkafka/librdkafka_darwin.a  -lm -lsasl2 -ldl -lpthread
```
Comment (mhowlett) [inline kafka/build_darwin.go:9]: i think this is statically linked now in the bundle, right?

File: kafka/librdkafka/rdkafka.h [hunk 2]
```diff
@@ -55,13 +55,16 @@ extern "C" {
 #endif
 #endif
 
-#ifdef _MSC_VER
+#ifdef _WIN32
```
Comment (mhowlett) [inline kafka/librdkafka/rdkafka.h:58]: i assume this file is copied direct from librdkafka.

**Review by mhowlett — APPROVED** [review]

File: kafka/librdkafka/rdkafka.h (hunk 2 above)
Comment (edenhill) [reply kafka/librdkafka/rdkafka.h:58]: Yep

File: kafka/build_darwin.go (hunk 1 above)
Comment (edenhill) [reply kafka/build_darwin.go:9]: Huhm, no, this is due to changes with the .pc file.
Good catch!

**Comment by edenhill** [comment]
I'll regenerate this with zlib fixed.

**Pull request closed without merging**